components/angular-tomitribe-tags/src/tags.controller.ts

```diff
@@ -50,7 +50,13 @@ export class TagsController {
 
                     const previousArray = $scope.availableTags && $scope.$$pagingState ? $scope.availableTags : [];
 
-                    $scope.availableTags = tagConfigurer.sortFn(_.union(previousArray, tagsFromServer));
```

Comment:
Are you sure you want to remove sorter function?

Reply:
Not at all. Big typo
Nice catch, thanks @Dexmaster